alembic/versions/9194fb66b4ea_add_doc_url_and_author_to_.py

```diff
@@ -0,0 +1,33 @@
+"""Add doc_url and author to BoundRegistryAction
+
+Revision ID: 9194fb66b4ea
+Revises: f2e840c9fb2c
+Create Date: 2025-01-06 20:10:46.871277
+
+"""
+import sqlmodel
```

Comment:
```suggestion
import sqlmodel.sql.sqltypes
```
Linter usually complains about this